app/main.py

```diff
@@ -1,2 +1,99 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+INITIAL_CAPACITY = 8
+RESIZE_THRESHOLD = 2 / 3
+CAPACITY_MULTIPLIER = 2
```

Comment:
You use this varibable only once. Do you really need it?
Also these constants belong to Dictionary implementation so they should be defined as class attrs